src/Miunie.Discord/MiunieDiscord.cs

```diff
@@ -1,23 +1,67 @@
 using System.Threading.Tasks;
+using DSharpPlus;
+using DSharpPlus.CommandsNext;
 using Miunie.Core;
+using Miunie.Discord.Configuration;
 
 namespace Miunie.Discord
 {
     public class MiunieDiscord : IDiscord
     {
-        private DSharpClient _client;
-        private CommandHandler _commandHandler;
+        private DiscordClient _discordClient;
+        private CommandsNextModule _commandsNextModule;
+        private IBotConfiguration _botConfiguration;
 
-        public MiunieDiscord(DSharpClient client, CommandHandler commandHandler)
+        public MiunieDiscord(IBotConfiguration botConfiguration)
         {
-            _client = client;
-            _commandHandler = commandHandler;
+            _botConfiguration = botConfiguration;
         }
 
         public async Task RunAsync()
         {
-            await _client.ConnectAsync();
-            await _commandHandler.InitializeAsync();
+
+            await InitializeDiscordClientAsync();
+
+            await InitializeCommandsNextModuleAsync();
+
+        }
```

Comment:
Can we get rid of the empty line 22 and 27? Just so it's more consistent with the rest of our code? :slightly_smiling_face: